src/overlays/actors/ovl_En_Ta/z_en_ta.c

```diff
@@ -1188,15 +1294,15 @@ s32 EnTa_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* po
 }
 
 void EnTa_PostLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot, void* thisx) {
-    static Vec3f D_80B16E7C = {
+    static Vec3f headOffset = {
         1100.0f,
         1000.0f,
         0.0f,
     };
```

Comment:
if you remove the comma on the last line and re run the formatter it will put it all in one line which looks a bit nicer in this context